### PR TITLE
chore: created a migration file that will create the entire database …

### PR DESCRIPTION
**Pull request by CurlyBracketEffect**
…from scratch

### CODE REVIEW

File: packages/server/migrations/1550779717609_full-db-migration.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      "id" SERIAL PRIMARY key,
+      "name" text UNIQUE NOT NULL,
+      "is_active" BOOLEAN NOT NULL,
+      "owner_id" INTEGER REFERENCES users(id) NOT NULL
+    );
+  `),
+  //4. Organizations_Users Table
+  pgm.sql(`
+    CREATE TABLE "foostown"."organizations_users" (
+      "organization_id" INTEGER  REFERENCES organizations (id) NOT NULL,
```
Comment (VPaulger) [inline packages/server/migrations/1550779717609_full-db-migration.js]: Extra space between INTEGER and REFERENCES

File: packages/server/migrations/1550779717609_full-db-migration.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    );
+  `),
+  //10. Elimination_Nodes Table
+  pgm.sql(`
+  CREATE TABLE "foostown"."elimination_nodes" (
+    "id" SERIAL PRIMARY KEY,
+    "round" INTEGER NOT NULL,
+    "position" INTEGER NOT NULL,
+    "tournament_id" INTEGER REFERENCES tournaments(id) NOT NULL,
+    "match_id" INTEGER  REFERENCES matches(id) NOT NULL,
```
Comment (VPaulger) [inline packages/server/migrations/1550779717609_full-db-migration.js]: Extra space between INTEGER and REFERENCES

File: packages/server/migrations/1550779717609_full-db-migration.js (hunk 1 above)
Comment (CurlyBracketEffect) [reply packages/server/migrations/1550779717609_full-db-migration.js]: fixed

File: packages/server/migrations/1550779717609_full-db-migration.js (hunk 2 above)
Comment (CurlyBracketEffect) [reply packages/server/migrations/1550779717609_full-db-migration.js]: Fixed

**Pull request merged**